model/version.go

```diff
@@ -307,9 +307,9 @@ func VersionGetHistory(versionId string, N int) ([]Version, error) {
 }
 
 func GetVersionsWithOptions(projectName string, opts GetVersionsOptions) ([]Version, error) {
-	projectId, err := FindIdForProject(projectName)
+	projectId, err := GetIdForProject(projectName)
 	if err != nil {
-		return nil, errors.Wrapf(err, "error finding id for project '%s'", projectName)
+		return nil, errors.WithStack(err)
```

Comment:
I would prefer the wrapped error since it's returned in a public API. It'll be weird/annoying to have a stack in a response

Reply:
I'll just return err then cause the wrap is still redundant